ovs-p4rt/sidecar/ovsp4rt.cc

```diff
@@ -352,7 +358,7 @@ void PrepareFdbTableEntryforV4VxlanTunnel(
           param->set_param_id(GetParamId(
               p4info, L2_FWD_TX_TABLE_ACTION_POP_VLAN_SET_VXLAN_UNDERLAY_V4,
               ACTION_PARAM_TUNNEL_ID));
-          param->set_value(EncodeVniValue(learn_info.tnl_info.vni));
+          param->set_value(EncodeTunnelId(learn_info.tnl_info.vni));
```

Comment:
Why change it from `EncodeVniValue` to `EncodeTunnelId`?? The member variable that is part of the name defined in `struct tunnel_info` is called `vni`. 

It would be better and cleaner to keep the name consistent instead of introducing new naming

Reply:
Because the field it's being encoded for is named `tunnel_id`, not `vni`, and the value is 20 bits, not 24.

Each use of the `vni` input field has its own semantics, which are related to the field in the P4Runtime request to which the value is written.

Using a purpose-specific function allows me to encapsulate the semantics (the high byte is ANDed with 0x0F instead of 0xFF). The function name alerts the reader to the different semantics.

I'm being consistent by using `EncodeTunnelId` for all 20-bit tunnel ID values and `EncodeVniValue` for all 24-bit VNI values, and I'm avoiding false equivalence by not using the same encoding function in both cases.